docs/cmdline-opts/ipfs-gateway.d

```diff
@@ -0,0 +1,36 @@
+c: Copyright (C) 2022, Mark Gaiser, <mark.gaiser@protocol.ai>
+SPDX-License-Identifier: curl
+Long: ipfs-gateway
+Arg: <URL>
+Help: Defines the gateway to handle IPFS protocol requests.
+Added: 7.87.0
+See-also: help manual
```

Comment:
I don't think either of these two options seem related to this. I think maybe `proxy` is one that is closest in functionality, even if it isn't for ipfs.

(Although of course, curl *can* do this IPFS handling over http proxy)

Reply:
"See-also" makes 0 sense. It's only there because it must be there. I'm fine changing it to whatever is acceptable, anything you put there won't be useful at all.

Reply:
What do you mean with it makes no sense? This is documentation. We help users by mentioning and referring to other options that the user *might* also (or sometimes rather) want to take a look at when they read about this option. This helps users find what they look for. We want to help our users.

Sure, it's sometimes hard to imagine what the best reference is, but saying that it makes no sense scares me.

Reply:
How about https://curl.se/docs/mqtt.html ? It is an example of non-http scheme.